internal/common/types.go

```diff
@@ -94,10 +81,7 @@ type DeviceInfo struct {
 
 // LoggingInfo is a struct which contains logging specific configuration settings.
 type LoggingInfo struct {
-	// EnableRemote defines whether to use Logging Service
-	EnableRemote bool
-	// File is the pathname of a local log file to be created.
-	File string
+	bootstrapConfig.LoggingInfo
```

Comment:
Same issue with embedding bootstrapConfig.LoggingInfo. Remove this local version and just use bootstrapConfig.LoggingInfo directly